vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -146,8 +146,8 @@ void call(Map parameters = [:], body) {
         if (!parameters.containerMap) {
             configHelper.withMandatoryProperty('dockerImage')
             config.containerName = 'container-exec'
-            config.containerMap = ["${config.get('dockerImage')}": config.containerName]
-            config.containerCommands = config.containerCommand ? ["${config.get('dockerImage')}": config.containerCommand] : null
+            config.containerMap = ["${config.get('dockerImage')}".toString(): config.containerName]
```

Comment:
We have a `java.lang.String` (config.dockerImage), we convert it into a `GString`(the double quotes) and change it back into a `java.lang.String` (the `toString). Seems to be more complicated that needed. What about this:

```
config.containerMap = [(config.get('dockerImage')): config.containerName]
```

or maybe even simpler:

```
config.containerMap = [(config.dockerImage): config.containerName]
```

We need the brackes surrounding the key, since we get compilation problems otherwise. But that seems to be more straight-forards compared to having `java.lang.String`, changing to `GString` and changing it back afterwards.

[Q]: Why are you sure to have a `java.lang.String` in the config map?
[A]: see [here](https://github.com/SAP/jenkins-library/blob/f3f4c741bea0513cdad77ebedf44552628f21e6d/src/com/sap/piper/ConfigurationHelper.groovy#L130)

Reply:
I assumed there was a reason that it was wrapped in a GString first (e.g. some type conversion, etc.), if that is not the case we can simplify it of course.